test/controllers/charges_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ChargesControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
ok.. I need to test it then.. I'm not amazing at testing but really need to improve my Rspec skills